ros_buildfarm/templates/devel/devel_job.xml.em

```diff
@@ -122,6 +122,10 @@ if pull_request:
         '# sleep to give python time to startup',
         'sleep 1',
         '',
+        '# define the build tool arguments',
+        'export build_tool_args="' + (build_tool_args or '') + '"',
+        'export build_tool_test_args="' + (build_tool_test_args or '') + '"',
```

Comment:
The extra round trip as an environment variable might be problematic if arguments contain double quotes themselves. Maybe they need to be escaped here?

Reply:
52757ec